docs/user-preferences.md

```diff
@@ -0,0 +1,67 @@
+# User Preferences  
+
+When the user configures the player according to his or her preference (e.g., starting up the player in mute), these user preferences (i.e., actions that are triggered by the user) create a *configurable value*,  which is saved in the browser's local storage. The player then uses those saved values to enable the user to retain the last choices they made when playing media.
+<br>For example, if the last played media was muted, the next media will begin muted also, even if the user refreshed the page.
+
+## Saved User Preference Values
+
+The following values are saved under user preferences:
+* Muted
+* Volume
+* Audio language track
+* Text language track (i.e., captions)
+* Captions text style (this is the style of the text used in the captions)
+
+## Workflow  
+
+When a player instantiates, a partial configuration is created internally by the `StorageManager` and supplied to the player using the saved values found in the local storage.
```

Comment:
I just want to be sure about the word instantiate - this is when a new instance occurs right? Not when the player starts running...?

Reply:
Yes - when a new instance occurs